src/fhirtypes/ElementDefinition.ts

```diff
@@ -633,9 +631,19 @@ export class ElementDefinition {
       this.checkSumOfSliceMins(max);
       // Check that new max >= every individual child max
       const slices = this.getSlices();
-      const overMaxChild = slices.find(child => child.max === '*' || parseInt(child.max) > maxInt);
-      if (!isUnbounded && overMaxChild) {
-        throw new InvalidMaxOfSliceError(overMaxChild.max, overMaxChild.sliceName, max);
+      const overMaxChildren: string[] = [];
+      slices.forEach(child => {
+        if (child.max === '*' || parseInt(child.max) > maxInt) {
+          child.max = max;
+          overMaxChildren.push(child.sliceName);
+        }
+      });
+      if (!isUnbounded && overMaxChildren.length > 0) {
+        logger.warn(
+          `Max of element ${this.id} is < max of its slices. The max of slice${
+            overMaxChildren.length > 1 ? 's:' : ':'
+          } ${overMaxChildren.join(', ')} has been reduced to ${max}`
+        );
```

Comment:
Also, I think the message would read awkwardly for multiple slices.  It would look like:
> Max of element Observation.category is < max of its slices. The max of slice: FooSlice, BarSlice has been reduced to 1

I think maybe it would read better for singular and plural if it was:
> At least one slice of Observation.category has a max greater than the overall element max. The max of the following slice(s) has been reduced to match the max of Observation.category: FooSlice, BarSlice

Since that uses `slice(s)` you don't need to have variable text.  It works singular or plural.  What do you think?  Does that read better to you?

Reply:
That reads _much_ better to me. I'll make that change!

Reply:
Addressed in commit `706d343`.